pinot-core/pom.xml

```diff
@@ -165,6 +165,10 @@
       <artifactId>annotations</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+    </dependency>
```

Comment:
Used in pinot-core/src/test/java/org/apache/pinot/queries/HistogramQueriesTest.java to improve error catching, test legibility and error message